cmd/tink-worker/action.go

```diff
@@ -24,32 +24,30 @@ var (
 )
 
 const (
-	errImagePull       = "failed to pull image: %s"
-	errCreateContainer = "failed to create container: %v"
-	errRemoveContainer = "failed to remove container: %v"
-	errRunContainer    = "failed to run container"
-	errFailedToWait    = "failed to wait for completion of action: %v"
-	errFailedToRunCmd  = "failed to run on-timeout command: %v"
+	errCreateContainer = "failed to create container"
+	errRemoveContainer = "failed to remove container"
+	errFailedToWait    = "failed to wait for completion of action"
+	errFailedToRunCmd  = "failed to run on-timeout command"
 
 	infoContainerStatus   = "on %s container status: %v"
-	infoContainerCreated  = "container created: %v"
-	infoContainerRemoved  = "container removed with status: %s"
+	infoContainerCreated  = "container created"
+	infoContainerRemoved  = "container removed with status"
 	infoWaitFinished      = "wait finished for failed or timeout container"
-	infoExitStatus        = "action container exit status code: %v"
-	infoRemoveContainer   = "starting to remove container: %s"
-	infoStartingContainer = "starting container with cmd: %s"
-	infoRunContainer      = "running container with id: %v"
+	infoExitStatus        = "action container exit status code"
+	infoRemoveContainer   = "starting to remove container"
+	infoStartingContainer = "starting container with cmd"
+	infoRunContainer      = "running container with id"
 )
 
-func executeAction(ctx context.Context, action *pb.WorkflowAction, wfID string) (string, pb.ActionState, error) {
+func executeAction(ctx context.Context, action *pb.WorkflowAction, wfID string) (pb.ActionState, error) {
 	l := logger.With("workflowId", wfID, "workerId", action.GetWorkerId(), "actionName", action.GetName())
```

Comment:
But now you've dropped the `action.GetImage()` from being logged which is probably very useful context to have.